vembrane/representations.py

```diff
@@ -270,33 +253,30 @@ def __init__(
         # Then, in the case of `number == "A"`, the value tuples only have one entry,
         # so that the value can be accessed directly and need not be accessed via
         # an index operation.
-        self._numbers = {
-            kind: {
-                record.get("ID"): overwrite_number.get(kind, {}).get(record.get("ID"))
-                or record.get("Number")
-                for record in header.records
-                if record.type == kind
-            }
-            for kind in set(r.type for r in header.records)
-        }
+        # self._numbers = {
+        #     category: {
+        #         record_name: overwrite_number.get(category, {}).get(record_name)
+        #         or record.get("Number")
+        #         for record_name, record in category_records.items()
+        #     }
+        #     for category, category_records in header.records.items()
+        # }
```

Comment:
This is now handled elsewhere and can be removed, correct?

Reply:
yes